### PR TITLE
Replaced urllib2 with requests. Ends up cleaner in my opinion

### PR DESCRIPTION
**Pull request by jorourke**
There is no doubt more that can be removed. I had to munge the "smooshy" script a bit so please feel free to edit that. I'm pretty new to fork/pull request process to please let me know any advice.

### CODE REVIEW

File: bin/smooshy [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/Users/james/.virtualenvs/smooshy/bin/python
```
Comment (josegonzalez) [inline bin/smooshy]: Revert this change please :)

**Comment by josegonzalez** [comment]
I actually don't like this. I like that this doesn't need external deps - hell, we can probably remove simplejson - and catching an extra exception could probably still be done with the older code.

Let me take a look at it when it's not 4am and I'll get back to you with better feedback though :)

**Comment by jorourke** [comment]
I would have dug a bit deeper but the smoosh.it "api" isn't very verbose in it's responses. It seems there's a filesize limit or timing out on bigger files.

**Comment by jorourke** [comment]
There's no doubt in my mind though that requests is far better than using urllib2 and also removed the need for multipart_handler.py while the active community around requests can manage those things.

**Comment by josegonzalez** [comment]
Closing as yahoo shutdown smushy :(
